xclim/indices/_agro.py

```diff
@@ -628,11 +629,17 @@ def dry_spell_frequency(
 
 @declare_units(pr="[precipitation]", thresh="[length]")
 def dry_spell_total_length(
-    pr: xarray.DataArray, thresh: str = "1.0 mm", window: int = 3, freq: str = "YS"
+    pr: xarray.DataArray,
+    thresh: str = "1.0 mm",
+    window: int = 3,
+    op: str = "sum",
+    freq: str = "YS",
+    start_date: str = "",
+    end_date: str = "",
 ) -> xarray.DataArray:
     """
-    Return the total number of days in dry periods of n days and more, during which the accumulated precipitation
-    on a window of n days is under the threshold.
+    Return the total number of days in dry periods of n days and more, during which the maximum or accumulated
+    precipitation on a window of n days is under the threshold.
```

Comment:
I suggest removing the reference to *n* in the title, and add a paragraph below to include details on the algorithm used.

Reply:
Also add references to papers or reports defining or using this algo.